code-coverage/lib/vitestProvider.ts

```diff
@@ -15,10 +13,12 @@ const customCoverageProviderModule: CoverageProviderModule = {
 				return ctx!.config.coverage;
 			},
 			async clean(clean?: boolean) {
+				const {default: setup} = await import('./setup');
```

Comment:
The two imports (setup and reportCoverage) could be moved to line 7, before the return.

Reply:
Indeed, that would be possible as well.